packages/components/src/components/Select/Select.tsx

```diff
@@ -63,7 +68,7 @@ export function Select<T extends object>({
           </Button>
           {description && <Text slot="description">{description}</Text>}
           <FieldError>{errorMessage}</FieldError>
-          <Popover>
+          <Popover {...popOverProps}>
```

Comment:
Is the weird casing deliberate? I think it should be either  `PopoverProps` or `popoverProps` as ["popover" is a single word](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API).

```suggestion
          <Popover {...popoverProps}>
```